src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -60,7 +80,10 @@ public Double averageSumOdd(List<Integer> numbers) {
      * Задача: Выбрать мужчин-военнообязанных (от `fromAge` до `toAge` лет)</p>
      **/
     public List<People> manSelectByAge(List<People> peopleList, int fromAge, int toAge) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(x -> x.getSex() == People.Sex.MAN)
```

Comment:
@AndreyDMA it's better to use more meaningful variables. If you have a Collection of People - it's better to write _p_ instead of _x_
```suggestion
                .filter(p -> x.getSex() == People.Sex.MAN)
```

Reply:
fixed